__tests__/skpm/.babelrc

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
why do you need this file?

Reply:
I pulled it from before the TypeScript refactor. I believe `@babel/plugin-transform-modules-commonjs` is needed for `@skpm/test-runner`? `@skpm/test-runner@0.4.1` has this plugin, but without `"noInterop": true`.

Without the `.babelrc` file, the tests give this error:

```bash
undefined is not an object (evaluating 'module.exports.tests = __skpm_tests__')
```